src/utils/axiosClient.ts

```diff
@@ -1,16 +1,14 @@
-// axios docs https://axios-http.com/docs/intro
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import axios from 'axios';
 
 const instance = axios.create({
   baseURL: 'https://mate.academy/students-api',
```

Comment:
Add a comment above line 6 to provide context about the axios instance and its purpose.